package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "arsenal",
   "engines": {
-    "node": ">=16"
+    "node": ">=10 <12"
```

Comment:
@nicolas2bert @naren-rajendran what do you think of limiting the node version until we are safe with 16?

Reply:
Let's remove the `engines` section until we have a clear defined supported node version (ie >= 16).
We would also need to specify the supported version in the respective project `README.md` .

Reply:
Are you sure we should remove it entirely? That would effectively mean Arsenal supports any node version.

Reply:
I like the idea of locking the supporting node version, until we fix the issues.

Reply:
I meant to revert the change without changing the engine for now. 
We would update it once we have a clear defined supported node version. But if you feel like adding supported versions, go for it.

Reply:
After seeing vault work on node 16 I will just use >=10 for now. One moment.